Source/Core/VideoBackends/OGL/Render.cpp

```diff
@@ -580,8 +597,21 @@ Renderer::Renderer()
 	}
 
 	glGetIntegerv(GL_MAX_SAMPLES, &g_ogl_config.max_samples);
-	if (g_ogl_config.max_samples < 1 || !g_ogl_config.bSupportsMSAA)
+	if (g_ActiveConfig.iMultisampleMode != MULTISAMPLE_SSAA_4X)
```

Comment:
I think this code needs to be re-executed on config change. Else use another integer for max SSAA samples.